### DISCUSSION TITLE
Bug: scrolling broken/missing in network graphs

### DISCUSSION THREAD

**Discussion by readnotify** [community/community#57307] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Someone used the wrong CSS element for these:

https://github.com/imartinez/privateGPT/network

they don't scroll using my mouse scroll-wheel or two-finger-swipe on my trackpad, and they don't have scrollbars to drag with a mouse.

Yes - I'm aware someone implemented 2 different non-standard ways to scroll them (click and drag, and keyboard hooks) - that's my point: non-standard.  Cute and unusual tricks probably don't belong in place of normal scroll operations that everyone knows and understands already...

**Comment by emilburganov** [comment]
This is because of what is implemented on canvas
